.github/workflows/post-merge-tasks.yml

```diff
@@ -0,0 +1,52 @@
+name: Post-merge tasks
+on:
+  push:
+    branches:
+      - master
+jobs:
+  update-release-draft:
+    runs-on: ubuntu-latest
+    name: "Release Drafter"
+    steps:
+      - uses: release-drafter/release-drafter@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  php:
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    name: "Coverage: Nextcloud PHP ${{ matrix.php-versions }}"
+    strategy:
+      matrix:
+        nextcloud: ['v21.0.0RC1']
```

Comment:
since the [`stable21`](https://github.com/nextcloud/server/tree/stable21) branch is available, we might as well use it, right?